ui/lib/core/addon/components/json-editor.hbs

```diff
@@ -10,6 +10,18 @@
         </label>
         <ToolbarActions>
           {{yield}}
+          {{#if @example}}
+            <button
+              type="button"
+              class="toolbar-link"
+              disabled={{not @value}}
+              {{on "click" this.restoreExample}}
+              data-test-restore-example
+            >
+              Restore example
+              <Icon @name="reload" />
+            </button>
+          {{/if}}
```

Comment:
An emerging pattern seems to be displaying examples in the code editor that can be restored so I decided to add it into the component.